[2022-09-06T15:09:22.440+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_hooks_v02.postgres_to_s3 scheduled__2022-05-23T00:00:00+00:00 [queued]>
[2022-09-06T15:09:22.457+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_hooks_v02.postgres_to_s3 scheduled__2022-05-23T00:00:00+00:00 [queued]>
[2022-09-06T15:09:22.458+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-06T15:09:22.458+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-06T15:09:22.458+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-06T15:09:22.478+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-05-23 00:00:00+00:00
[2022-09-06T15:09:22.484+0000] {standard_task_runner.py:52} INFO - Started process 1201 to run task
[2022-09-06T15:09:22.488+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v02', 'postgres_to_s3', 'scheduled__2022-05-23T00:00:00+00:00', '--job-id', '614', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmp3x99rxpm', '--error-file', '/tmp/tmpcgztkxgy']
[2022-09-06T15:09:22.488+0000] {standard_task_runner.py:80} INFO - Job 614: Subtask postgres_to_s3
[2022-09-06T15:09:22.609+0000] {task_command.py:371} INFO - Running <TaskInstance: dag_with_postgres_hooks_v02.postgres_to_s3 scheduled__2022-05-23T00:00:00+00:00 [running]> on host 45495cb94d52
[2022-09-06T15:09:22.715+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=James
AIRFLOW_CTX_DAG_ID=dag_with_postgres_hooks_v02
AIRFLOW_CTX_TASK_ID=postgres_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-05-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-23T00:00:00+00:00
[2022-09-06T15:09:22.716+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-06T15:09:22.728+0000] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-09-06T15:09:22.734+0000] {dag_with_postgres_hooks.py:30} INFO - save orders data in text file dags/get_orders_20220523.txt
[2022-09-06T15:09:22.735+0000] {python.py:173} INFO - Done. Returned value was: None
[2022-09-06T15:09:22.753+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v02, task_id=postgres_to_s3, execution_date=20220523T000000, start_date=20220906T150922, end_date=20220906T150922
[2022-09-06T15:09:22.800+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-06T15:09:22.839+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
